app/views/users/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: user) do |form| %>
+  <% if user.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+        <% user.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div>
+    <%= form.label :photo, style: "display: block" %>
+    <%= form.text_field :photo %>
+  </div>
+
+  <div>
+    <%= form.label :bio, style: "display: block" %>
+    <%= form.text_field :bio %>
+  </div>
+
+  <div>
+    <%= form.label :posts_counter, style: "display: block" %>
+    <%= form.text_field :posts_counter %>
+  </div>
```

Comment:
- [ ] The post counter should count the number of posts a user has and not be input while creating a user. Kindly remove the field, and add the post counter in the controller instead